contrib/postgres_proxy/filters/network/test/postgres_integration_test.cc

```diff
@@ -322,8 +322,9 @@ class UpstreamSSLBaseIntegrationTest : public PostgresBaseIntegrationTest {
     static auto* client_stats_store = new Stats::TestIsolatedStoreImpl();
     Network::DownstreamTransportSocketFactoryPtr tls_context =
         Network::DownstreamTransportSocketFactoryPtr{
-            new Extensions::TransportSockets::Tls::ServerSslSocketFactory(
-                std::move(cfg), *tls_context_manager, *(client_stats_store->rootScope()), {})};
+            Extensions::TransportSockets::Tls::ServerSslSocketFactory::create(
+                std::move(cfg), *tls_context_manager, *(client_stats_store->rootScope()), {})
+                .value()};
```

Comment:
Here and elsewhere, https://abseil.io/tips/181 recommends `*` instead of `.value()` (treating a StatusOr a lot like a unique_ptr)

Reply:
will do